gtest_parallel_mocks.py

```diff
@@ -141,3 +147,41 @@ def __init__(self, test_id, execution_number, test_data):
 
   def run(self):
     pass
+
+
+class OptionsMock(object):
+  def __init__(self, failed=False, gtest_also_run_disabled_tests=False,
```

Comment:
Is there any way that we can assert that these options are the same as the default `optparse` options? I don't want us to maintain two copies of the default values without enforcing equality.

Maybe the optparse arguments could be broken out of main?